addon-test-support/did-intersect-mock.js

```diff
@@ -39,22 +39,26 @@ class MockIntersectionObserver {
   /**
    * Force a single element to enter the viewport
    * @param {String} el - a DOM selector string
+   * @param {object} [state] Additional state to be passed as the IntersectionObserverEntry
```

Comment:
Lets type this as `IntersectionObserverEntry` instead of `object`?

Reply:
fixed